### ISSUE TITLE
Empty arrays/objects should not be expandable

### ISSUE THREAD

**Issue by juancampa** [dmackdev/egui_json_tree#5]
<img width="295" alt="Screenshot 2023-09-06 at 11 34 40 PM" src="https://github.com/dmackdev/egui_json_tree/assets/1410520/705c3d34-0b76-4e9f-8eaa-e7de7e3ee629">

I suggest that these are not rendered as "collapsing" and don't show the `...`.

**Comment by dmackdev** [comment]
I think for consistency in JSON representation, empty arrays/objects should still show the collapsing icon and be expandable. I think showing `[]` and `{}` for empty  arrays/objects would make sense.
Other JSON tree visualisers also follow this combined approach.
Would you like to raise a PR to remove the `...` for empty arrays/objects?

**Comment by juancampa** [comment]
Fair enough, I see that being better when there are more keys, some empty and some not.

Interestingly, Chrome mixes both representations in the JS console.
![Screenshot 2023-09-07 at 8 20 46 PM](https://github.com/dmackdev/egui_json_tree/assets/1410520/5ee1edfb-93ff-4e77-b719-0ee7df45a3f5)

I'll make a PR with my changes
